### DISCUSSION TITLE
Best way to translate (at Weblate.org) newly added phrases before the release?

### DISCUSSION THREAD

**Discussion by slrslr** [mifi/lossless-cut#2015] [General]
One can translate a LC into various languages: https://hosted.weblate.org/projects/losslesscut/#languages
Question: What is the best approach to translate newly added phrases on-time before the release of a new LC version?

Currently i am subscribed to a new release notifications here on a Github.

Today I went to above linked Weblate project page, clicked the Watching button on the top-right and then has been redirected to https://hosted.weblate.org/accounts/profile/#notifications , where i have scrolled down to "Notification settings" and clicked "Project: LosslessCut", in its "Translation notifications" section set "Daily digest" for "String is available for translation".

**Comment by mifi** [comment]
Good question. TBH I'm not sure.

> Today I went to above linked Weblate project page, clicked the Watching button on the top-right and then has been redirected to https://hosted.weblate.org/accounts/profile/#notifications , where i have scrolled down to "Notification settings" and clicked "Project: LosslessCut", in its "Translation notifications" section set "Daily digest" for "String is available for translation".

I think that sounds like a good way to receive a notification of new strings. New strings are added automaticlaly in weblate whenever I run `yarn scan-i18n`, which I try to do every now and then.

example commit:
https://github.com/mifi/lossless-cut/commit/04179320c33dbb8f97273cbf6ecaaae1170c7a9d
